app/views/ingredients/index.html.erb

```diff
@@ -0,0 +1,11 @@
+<h1>Ingredients</h1>
+
+<! We are using AR syntax here to grab all the ingredient objects, >
+  <! order them (by name - remember that is the name column on the ingredients table) >
+<! So when we do the `.each` on `@ingredients.order(:name)` we already have >
+  <! all of the ingredients from that table sorted alphabetically >
+
+<% @ingredients.order(:name).each do |ingredient| %>
+  <p> Name: <%= ingredient.name %></p>
+  <p> Cost: <%= ingredient.cost %></p>
+  <% end %>
```

Comment:
Doing `order` in the view would be considered an MVC infraction. You want to abstract this method to the model.

Just note that there was a specific format for how the ingredients names and cost were supposed to be listed. `Name: Cost`.